.github/workflows/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Run Rust tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Could you please use this action to setup rust?
https://github.com/chris-zen/kiro-synth/blob/master/.github/workflows/build.yaml#L40